h/static/styles/partials-v2/_masthead.scss

```diff
@@ -7,3 +7,7 @@
   padding-right: 15px;
   padding-top: 15px;
 }
+
+.masthead-logo {
+  color: $black;
```

Comment:
Presumably this should be the same color as the logo when it appears in the navbar (ie. a very dark grey)?